core/src/commonMain/kotlin/com/ustadmobile/core/view/PersonAccountEditView.kt

```diff
@@ -0,0 +1,33 @@
+package com.ustadmobile.core.view
+
+import com.ustadmobile.lib.db.entities.Person
+
+interface PersonAccountEditView: UstadEditView<Person> {
+
+    var firstPassword: String?
```

Comment:
Let's put these on an entity as per comment on presenter so this follows the pattern